test-cases/artifacts/ubuntu2204.yaml

```diff
@@ -0,0 +1,23 @@
+backtrace_decoding: false
+cluster_backend: 'gce'
+gce_image_db: 'https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2204-lts'
+gce_instance_type_db: 'n1-standard-2'
+gce_root_disk_type_db: 'pd-ssd'
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "spot"
+instance_provision_fallback_on_demand: true
+logs_transport: 'ssh'
+n_db_nodes: 1
+n_loaders: 0
+n_monitor_nodes: 0
+nemesis_class_name: 'NoOpMonkey'
+scylla_linux_distro: 'ubuntu-jammy'
+scylla_repo: 'http://downloads.scylladb.com/unstable/scylla/master/deb/unified/latest/scylladb-master/scylla.list'
```

Comment:
Tbh this just mirrors what we have for other ubuntu artifact test cases. But it does point to the repo that *should* have the latest master

Reply:
what we have in this parameter, for release branches? also pointing to latest master?

Reply:
Yup, it also points to master in branch-4.6 for example. We provide the actual param in the job params though, so that gets used instead of the test case config value.

Reply:
@fruch is right, please add a comment that this is something that is being override by the Jenkins job parameter.

Reply:
@roydahan I removed the line completely. I agree with @fruch that it would be better to fail fast if somehow the job doesn't provide the value. My runs work ok without this line.